include/drivers/led_strip.h

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) 2017 Linaro Limited
  *
  * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileComment: IEC-61508-SIL3
```

Comment:
Do you really need to safety certify this API and its drivers? This seems like overkill. I am opposed to this.

Reply:
The driver model was marked as in the safety scope in the pdf (see description) which I interpret as the API. Where as the drivers implementation since very often MCU specific is not (yet) and has to be done by the Zephyr OS safety application user.

Changing the driver API will very likely cause the drivers to be changed and re-certified (even though not by the project) by a safety user of Zephyr OS. Therefore I can understand why the driver model was marked as safety scope.
If all drivers and which driver, is subject to discussions and definitions.